tck/src/it/resources/application.conf

```diff
@@ -1,3 +1,4 @@
+cloudstate-tck.verify = ["Akka + Node.js", "Akka + Java", "Akka + Scala", "Akka + Go", "Akka + Kotlin"]
```

Comment:
I created this so that it is easier to specify which of the combinations one wants to run.

Reply:
Needs to be added to `native-image.conf` as well, otherwise the native TCK integration test won't run.

Reply:
@pvlugter Fixed.